### PR TITLE
update wenxin aboutus

### PR DESCRIPTION
**Pull request by wenxin-c**

### CODE REVIEW

**Comment by YongbinWang** [comment]
Fits the requirement required. Links work. LGTM.

File: docs/AboutUs.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-Display | Name | Github Profile | Portfolio 
---------|:----:|:--------------:|:---------:
-![](https://via.placeholder.com/100.png?text=Photo) | John Doe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Joe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Ron John | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | John Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Don Roe | [Github](https://github.com/) | [Portfolio](docs/team/johndoe.md)
+| Display                                             |    Name     |            Github Profile             |             Portfolio             |
+|-----------------------------------------------------|:-----------:|:-------------------------------------:|:---------------------------------:|
+| ![](./team/wenxin.jpg)                              | Chen Wenxin | [Github](https://github.com/wenxin-c) | [Portfolio](docs/team/wenxin.md)  |
```
Comment (haoyangw) [inline docs/AboutUs.md:5]: Correct markdown formatting, nice

File: docs/team/wenxin.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Wenxin - Project Portfolio Page
```
Comment (haoyangw) [inline docs/team/wenxin.md:1]: Updated the header as well, good job!

**Review by YongbinWang — APPROVED** [review]
Satisfied requirements, fixed personal md file. LGTM.

**Review by haoyangw — APPROVED** [review]
All changes work as intended to add a new profile page, no issues found. LGTM!

**Pull request merged**